input/gammacat/gamma_cat_columns.yaml

```diff
@@ -299,6 +299,41 @@
   unit: TeV cm-2 s-1
   description: Integrated energy flux between 1 TeV and 10 TeV error
   latex:
+- name: spec_phase_min
+  fmt:
+  dtype: float32
+  unit:
+  description: phase range minimum (e.g. orbital or pulsar phase)
+  latex:
+- name: spec_phase_max
+  fmt:
+  dtype: float32
+  unit:
+  description: phase range maximum (e.g. orbital or pulsar phase)
+  latex:
+- name: spec_mjd_min
+  fmt:
+  dtype: float32
+  unit: days
+  description: MJD range minimum
+  latex:
+- name: spec_mjd_max
+  fmt:
+  dtype: float32
+  unit:
+  description: MJD range maximum
+  latex:
+- name: observatory
+  dtype: str
+  fmt:
+  unit:
+  description: Observatory name
+ name: fileindex
```

Comment:
There's a missing minus here on this line `name: fileindex`.
That's the cause of the YAML parse error:
```
ParserError: while parsing a block collection
  in "a.yaml", line 2, column 1
expected <block end>, but found '<block mapping start>'
  in "a.yaml", line 331, column 2
```